SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -891,7 +902,7 @@ int32 USpatialNetDriver::ServerReplicateActors_ProcessPrioritizedActors(UNetConn
 					if (!InConnection->IsNetReady(0))
 					{
 						// We can bail out now since this connection is saturated, we'll return how far we got though
-						return FinalReplicatedCount;
+						return FinalReplicatedCount + FinalCreationCount;
```

Comment:
This is never executed since `IsNetReady` for `USpatialNetConnection` is always true.
Please remove this code and add a comment saying this bit was removed (and referencing the corresponding line of native net driver?)